### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by srknzl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     },
     "devDependencies": {
         "@types/node": "~10.17.60",
-        "@typescript-eslint/eslint-plugin": "~4.28.5",
-        "@typescript-eslint/parser": "~4.28.5",
+        "@typescript-eslint/eslint-plugin": "~4.29.0",
+        "@typescript-eslint/parser": "~4.29.0",
```
Comment (mdumandag) [inline package.json:18]: These guys are releasing new versions non-stop :smile:

**Review by mdumandag — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (srknzl) [reply package.json:18]: They have a strange release cycle, they release every week even if nothing changes.

Comment (mdumandag) [reply package.json:18]: maybe we need to increase the period of the new version check to something like 2-4 weeks then?

Comment (srknzl) [reply package.json:18]: Maybe. Are these prs too much? :D If so, we should

Comment (srknzl) [reply package.json:18]: Do you see any way to set it 2 weeks etc. 

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#scheduleinterval

only weekly, monthly or daily is available

Comment (mdumandag) [reply package.json:18]: No, I am not aware of such a method. Personally, I would set it to monthly if these are the only options, but I am leaving the decision to you.

Comment (srknzl) [reply package.json:18]: I don't think we have critical dependencies that needs to be updated immediately, we have only `long.js` which should be mature already. As you can see from dependencies update PRs, it is not updated often. if our repo settings enable security warnings about dependencies, it should be enough.

So I will make it monthly.

**Pull request merged**